v3/src/models/data/formula-fn-registry.ts

```diff
@@ -104,6 +154,58 @@ export const fnRegistry = {
     evaluate: (a: any, b: any) => !equal(a, b)
   },
 
+  abs: {
+    evaluate: numericFnFactory(Math.abs)
+  },
+
+  ceil: {
+    evaluate: numericFnFactory(Math.ceil)
+  },
+
+  combinations: {
+    evaluate: numericMultiArgsFnFactory(math.combinations, { numOfRequiredArgs: 2 })
+  },
+
+  exp: {
+    evaluate: numericFnFactory(Math.exp)
+  },
+
+  floor: {
+    evaluate: numericFnFactory(Math.floor)
+  },
+
+  frac: {
+    evaluate: numericFnFactory((v: number) => v - (v < 0 ? Math.ceil(v) : Math.floor(v)))
```

Comment:
Isn't this the same as:
```suggestion
    evaluate: numericFnFactory((v: number) => v - Math.trunc(v))
```
?

Reply:
It is, thanks. :)